files/default/neutron-ha-tool.py

```diff
@@ -737,10 +737,21 @@ def target_agent_list(agent_list, agent_type, exclude_agent_host):
     :param exclude_agent_host: hostname of agent to exclude from the list
```

Comment:
> Why the comment is in the line of code?

Because [github doesn't allow reviewing of commit messages](https://github.com/SUSE-Cloud/cookbook-openstack-network/pull/6#discussion_r88211081).

> Where do you want that I add the full link of the bug?

In the commit message, like I said.

> The bug is stated in the commit message, in such a way that can be listed in the changelog.

Yes, but the *full URL* is missing.

> I am not aware of this team policy etherpad

You are now, since I told you ;-)

> and I am not sure that OBS / IBS can track the full URL in the commit description.

It doesn't need to; it can track the `bsc#XXXXXX` number.

> I would prefer to add this bsc#XXXXX signature for convenience in the commit message.

I didn't ask you to drop the `bsc#XXXXXXX`.  In fact the etherpad requires that, and IMHO it's exactly right that you have already put the `bsc#XXXXXXX` in the first line of the commit message.  I was asking for the full URL to be included in the commit message body, in *addition* to that.  The justification is given in the etherpad.